pkg/cmd/dev/test.go

```diff
@@ -257,6 +257,26 @@ func (d *dev) test(cmd *cobra.Command, commandLine []string) error {
 		testTargets = append(testTargets, target)
 	}
 
+	// List of bazel integration tests that will fail when running `dev test pkg/...`
+	var integrationTests = map[string][2]string{
+		"pkg/acceptance":              {"acceptance_test", "`dev acceptance`"},
```

Comment:
Let's make the map value a struct rather than an array of two strings. This way we don't have to remember what the 0 and 1 elements of the array mean. (Haven't tested this, but the syntax should work.)

```
var integrationTests = map[string]struct { testName, commandToRun string } {
    "pkg/acceptance":              {"acceptance_test", "dev acceptance"},
   ....
}
```